### PR TITLE
feat: service-monitor event-gateway

### PR DESCRIPTION
**Pull request by bvdeenen**

### CODE REVIEW

File: helm/Chart.lock [hunk 1]
```diff
@@ -1,12 +1,15 @@
 dependencies:
 - name: postgresql
   repository: https://charts.bitnami.com/bitnami
-  version: 11.1.13
+  version: 11.1.29
```
Comment (ivan-p92) [inline helm/Chart.lock:4]: What happens if a user with an existing helm install would do an upgrade? Is there data loss due to postgres being reprovisioned?

**Review by ivan-p92 — APPROVED** [review]

**Comment by ivan-p92** [comment]
I noticed there are no new tarballs under *charts* for kafka and postgres. Is this intentional?

File: helm/Chart.lock (hunk 1 above)
Comment (bvdeenen) [reply helm/Chart.lock:4]: No, the `helm upgrade` shows how to use the existing postgres password

**Comment by bvdeenen** [comment]
I don't think these tarballs ever should have been in the repo.

**Pull request merged**